### PR TITLE
Fix PMD issues

### PR DESCRIPTION
**Pull request by rkondratowicz**
some of them...

### CODE REVIEW

File: config/pmd/ruleset.xml [hunk 1]
```diff
@@ -17,6 +17,7 @@
     <exclude name="MethodArgumentCouldBeFinal" />
     <exclude name="OnlyOneReturn" />
     <exclude name="TooManyStaticImports" />
+    <exclude name="ConfusingTernary" />
```
Comment (rkondratowicz) [inline config/pmd/ruleset.xml]: to allow `if(x != y) { ... }`

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/AuthServiceConfig.java [hunk 2]
```diff
@@ -39,14 +39,17 @@ public AuthTokenValidator tokenValidator(ServiceAuthorisationApi s2sApi) {
...
             }
 
+            @Override
             public void validate(String token, List<String> roles) {
                 throw new NotImplementedException();
             }
 
+            @Override
             public String getServiceName(String token) {
                 return "some_service_name";
```
Comment (aliveni) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/AuthServiceConfig.java]: Why are we overriding the above methods?

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/AuthServiceConfig.java]: methods were always overridden, just adding annotation to make it clear (as suggested by PMD)

Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/AuthServiceConfig.java]: oh, got it. didn't notice the interface earlier

**Pull request merged**